talk/C++Course.tex

```diff
@@ -222,4 +222,47 @@ \section*{Index}
   \listofgodbolt
 \end{frame}
 
+\section*{External Resources and Further Reading}
+
+\begin{frame}
+  \frametitle{Books}
+  \begin{thebibliography}{10}
+    \scriptsize
+    \beamertemplatebookbibitems
+
+    \bibitem{} A Tour of C++, Third Edition
+    \newblock  Bjarne Stroustrup, Addison-Wesley, Sep 2022
+    \newblock  ISBN-13: \href{https://learning.oreilly.com/library/view/a-tour-of/9780136823575/}{978-0136816485}
+
+    \bibitem{} Effective Modern C++
+    \newblock  Scott Meyers, O'Reilly Media, Nov 2014
+    \newblock  ISBN-13: \href{https://learning.oreilly.com/library/view/effective-modern-c/9781491908419/}{978-1-491-90399-5}
+
+    \bibitem{} Clean Architecture
+    \newblock  Robert C. Martin, Pearson, Sep 2017
+    \newblock  ISBN-13: \href{https://learning.oreilly.com/library/view/clean-architecture-a/9780134494272/}{978-0-13-449416-6}
+
+    \bibitem{} The Art of UNIX Programming
+    \newblock  Eric S. Raymond, Addison-Wesley, Sep 2002
+    \newblock  ISBN-13: \href{https://learning.oreilly.com/library/view/the-art-of/0131429019/}{978-0131429017}
+
+    \bibitem{} Introduction to Algorithms, 4th Edition
+    \newblock  T. H. Cormen, C. E. Leiserson, R. L. Rivest, C. Stein, Apr 2022
+    \newblock  ISBN-13: \href{https://catalogue.library.cern/literature/ktyga-1g862}{978-0262046305}
+  \end{thebibliography}
+\end{frame}
+
+\begin{frame}
+  \frametitle{Conferences}
+  \begin{itemize}
+    \item CppCon --- \href{https://cppcon.org/}{cppcon.org}
+    \item \cpp Now --- \href{https://cppnow.org/}{cppnow.org}
+    \item Code::Dive --- \href{https://codedive.pl/}{codedive.pl}
+    \item ACCU Conference --- \href{https://accu.org/}{accu.org}
+    \item Meeting \cpp\ --- \href{https://meetingcpp.com/}{meetingcpp.com}
+    \item See link below for more information
+          \url{https://isocpp.org/wiki/faq/conferences-worldwide}
+  \end{itemize}
```

Comment:
Good selection!

Reply:
Thanks! I think this may actually be better on a description list, now that I think about it...

Reply:
Actually, it looked terrible with definition list, so I will keep it as is.